ibis/backends/sqlite/client.py

```diff
@@ -412,3 +413,88 @@ def _table_from_schema(
     ) -> sa.Table:
         columns = self._columns_from_schema(name, schema)
         return sa.Table(name, self.meta, schema=database, *columns)
+
+    def insert(
```

Comment:
Did you check if this could be implemented in the base SQLAlchemy client, so it also works for Postgres and MySQL?

Reply:
I think it can be implemented. Should I do it in this PR itself? Also, this might require updating of test cases as well.

Reply:
yes pls do

Reply:
@datapythonista, @jreback please take a look now. Have added (moved and modified) this function to both the places relevant under the `AlchemyClient` class under the `backends` directory.